pkg/devfile/adapters/common/command.go

```diff
@@ -158,8 +186,11 @@ func ValidateAndGetPushDevfileCommands(data data.DevfileData, devfileBuildCmd, d
 	}
 
 	// If either command had a problem, return an empty list of commands and an error
-	if !isBuildCommandValid || !isRunCommandValid {
+	if !isInitCommandValid || !isBuildCommandValid || !isRunCommandValid {
 		commandErrors := ""
+		if initCmdErr != nil {
+			commandErrors += initCmdErr.Error()
```

Comment:
should we add a new line between each error so they are more readable? or maybe consider `errors.Wrap`?